masterlist.yaml

```diff
@@ -25749,72 +25749,107 @@ plugins:
     msg: [ *corrupt ]
 
 ###### Waalx Animals & Creatures (WAC) ######
-  - name: 'Waalx Animals & Creatures.esm'
-    url: [ 'http://tesalliance.org/forums/index.php?/files/file/1318-wac-waalx-animals-creatures/' ]
+  - name: '(Waalx Animals & Creatures|WAC( - .*)?|Natural_Habitat_by_Max_Tael)\.es[mp]'
+    url:
+      - link: 'https://tesalliance.org/forums/index.php?/files/file/1318-wac-waalx-animals-creatures/'
+        name: 'WAC: Waalx Animals & Creatures v1.0 Additional Files'
     clean:
-      - crc: 0x8A731536
-        util: 'TES4Edit v4.0.2i'
+      - crc: 0x3220E267
+        util: 'TES4Edit v4.0.4c'
```

Comment:
If this checksum does apply to multiple plugins, this regex is too broad. It'd be better to only apply it to the plugins it applies to.